### ISSUE TITLE
Record UUID of VM in addition to domain name

### ISSUE THREAD

**Issue by dgarth** [intelsdi-x/snap-plugin-collector-libvirt#37]
(Feature Request) It would help a lot identifying VMs quickly and conveniently - especially in an OpenStack environment - if the UUID was also recorded into the measurements instead of just the domain name.

**Comment by sandlbn** [comment]
@dgarth right now openstack id is reported as an uuid when nova config is set to true. In the new refactored version this is will be visible as nova_uuid tag. Can you check if this amount of tags is enough for you. 

https://github.com/sandlbn/snap-plugin-collector-libvirt/blob/a2d73d0ee8da8d694707426e65980c1829b5c8df/libvirt/domain.go#L153

**Comment by dgarth** [comment]
Yes, thanks, for me those would be more than enough.

In the current version the nova setting does not seem to work though. I set
```
"config": {
    "/libvirt": {
        "nova": true
    }
}
```
And there are no additional tags in the database (using InfluxDB as publisher).
Also, when adding another configuration option to the task, e.g.
```
"config": {
    "/libvirt": {
        "nova": true
    },
    "/intel/procfs": {
        "proc_path": "/proc"
    }
}
```
Snap panics:
```
 Received: PANIC: Can't add a new root namespace
goroutine 122304 [running]:
github.com/intelsdi-x/snap/vendor/github.com/codegangsta/negroni.(*Recovery).ServeHTTP.func1(0x7fb562117930, 0xc8206f5340, 0xc82004b180)
        /snapbuild/build/snap/src/github.com/intelsdi-x/snap/vendor/github.com/codegangsta/negroni/recovery.go:34 +0xe9
panic(0xb85b00, 0xc8202751a0)
        /gonative/go/src/runtime/panic.go:443 +0x4e9
github.com/intelsdi-x/snap/pkg/ctree.(*ConfigTree).Add(0xc820274d80, 0xc8203754d0, 0x1, 0x1, 0x7fb562115170, 0xc820275110)
        /snapbuild/build/snap/src/github.com/intelsdi-x/snap/pkg/ctree/tree.go:116 +0x946
github.com/intelsdi-x/snap/core/cdata.(*ConfigDataTree).Add(0xc820165b98, 0xc8203754d0, 0x1, 0x1, 0xc820275110)
        /snapbuild/build/snap/src/github.com/intelsdi-x/snap/core/cdata/tree.go:59 +0x77
github.com/intelsdi-x/snap/scheduler/wmap.(*CollectWorkflowMapNode).GetConfigTree(0xc8204b7810, 0x0, 0x0, 0x0)
        /snapbuild/build/snap/src/github.com/intelsdi-x/snap/scheduler/wmap/wmap.go:254 +0x447
github.co
```

**Comment by sandlbn** [comment]
@dgarth this will be also addressed in #36 . Please check this task manifest https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-libvirt/master/example/task-example.json

**Comment by sandlbn** [comment]
Fixed in #36
